templates/cisco_ios_show_hosts_summary.template

```diff
@@ -0,0 +1,12 @@
+Value DEFAULT_DOMAIN (\S+)
+Value NAME_SERVERS ([\S ]+)
```

Comment:
Can you change the `NAME_SERVERS` regex to **.+**?

Also, can you please change any static spacing within the **Start** state to either **\s*** or **\s+**?

After those are addressed, we can merge this.

Reply:
Thank you for pointing out the NAME_SERVERS.
I also updated the regex in the start state...